website/static/js/newAndNoteworthyPlugin.js

```diff
@@ -73,6 +79,10 @@ var NewAndNoteworthy = {
         };
     },
     view : function(ctrl) {
+        if (ctrl.errorLoading) {
```

Comment:
ctrl.errorLoading is not an m.prop. This will not refresh the page if the error issue is resolve. What I see is that you don't expect an error to resolve anyway, is that correct?